src/sass/pages/main-page/_hero.scss

```
@@ -0,0 +1,100 @@
+.main-hero {
+  padding-bottom: 40px;
+
+  @include tablet {
+    padding-bottom: 60px;
+  }
+  @include desktop {
+    padding-bottom: 120px;
+  }
+}
+.main-hero__container {
+  @include tablet {
+    width: 400px;
+  }
+  @include desktop {
+    width: 672px;
+  }
+}
+
+.main-hero__title {
+  text-align: center;
+  margin: 0 auto 32px;
+  font-family: 'Caviar Dreams', sans-serif;
+  @include font(18px, 400, 21px, null);
+  @media screen and (max-width: 1439px) {
+    max-width: 336px;
+  }
+
+  // font-size: 18px;
+  // font-weight: 21px;
+  @include tablet {
+    @include font(24px, 400, 28px, null);
+    margin-bottom: 60px;
+
+    // font-size: 24px;
+    // line-height: 28px;
+  }
+  @include desktop {
+    // max-width: 630px;
+    @include font(48px, 400, 56px, null);
+    // font-size: 48px;
+    // line-height: 56px;
+  }
+}
+.hero__after-title {
+  text-align: center;
+  font-family: 'Caviar Dreams', sans-serif;
```

```suggestion
  font-family: 'Caviar Dreams Regular', sans-serif;
```
По шрифтам: уже писал в чат))) 
Наши шрифты:
font-family: 'Caviar Dreams Regular';
font-family: 'Caviar Dreams Bold';
font-family: 'Cera Pro Light';
font-family: 'Cera Pro Regular';
font-family: 'Cera Pro Medium';
font-family: 'Cera Pro Bold';